content/en/docs/tasks/administer-cluster/declare-network-policy.md

```diff
@@ -66,98 +67,84 @@ pod/nginx-701339712-e0qfq   1/1           Running       0          35s
 
 ## Test the service by accessing it from another pod
 
-You should be able to access the new `nginx` service from other pods. To test, access the service from another pod in the default namespace. Make sure you haven't enabled isolation on the namespace.
+You should be able to access the new `nginx` Service from other Pods. To test, access the Service from another Pod in the `default` namespace. Make sure you haven't enabled isolation on the namespace.
 
-Start a busybox container, and use `wget` on the `nginx` service:
+You can start a busybox container to get a shell and use `wget` on the `nginx` Service:
 
 ```console
 kubectl run --generator=run-pod/v1 busybox --rm -ti --image=busybox -- /bin/sh
 ```
 
-```console
-Waiting for pod default/busybox-472357175-y0m47 to be running, status is Pending, pod ready: false
+In your shell, run the following command:
 
-Hit enter for command prompt
+```shell
+wget --spider --timeout=1 nginx
+```
 
-/ # wget --spider --timeout=1 nginx
+```none
 Connecting to nginx (10.100.0.16:80)
-/ #
+remote file exists
 ```
 
-## Limit access to the `nginx` service
+## Limit access to the `nginx` Service
 
-Let's say you want to limit access to the `nginx` service so that only pods with the label `access: true` can query it. To do that, create a `NetworkPolicy` that allows connections only from those pods:
+Let's say you want to limit access to the `nginx` Service so that only Pods with the label `access: true` can query it. You can create a NetworkPolicy to allow connections from Pods that have the `access: true` label:
```

Comment:
Possible clean up of Lines 90 -91:
To limit the access to the `nginx` service so that only Pods with the label `access: true` can query it, create a NetworkPolicy object as follows:

Reply:
Done.